File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height = 0;
    while (height < 1)
    {
        height = get_int("Height: "); // get_int deals with char/string inputs
    }
    for (int i = 1; i <= height; i++)
    {
        for (int j = 0; j < (height - i); j++)
        {
            printf(" ");
        }
        for (int k = 0; k < i; k++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: sentimental-mario-more/mario.py
import cs50

while True:
    n = cs50.get_int("Height: ")
    if n in range(1, 9):
        break

blank = n
for i in range(1, n+1):
    blank -= 1
    print(" " * blank, end="")
    print("#" * i, end="")
    print("  ", end="")
    print("#" * i)
